packages/admin-stories/src/docs/comet/admin/toolbar.stories.mdx

```diff
@@ -445,3 +449,42 @@ The Toolbar expands automatically if it needs more space
         </StackApiContext.Consumer>
     </Story>
 </Canvas>
+
+### FilterBar
+
+<Canvas>
+    <Story name="Filterbar">
+        {() => {
+            const Story = () => {
+                const filterApi = useTableQueryFilter({});
+                return (
+                    <Toolbar>
+                        <ToolbarTitleItem>Filterbar</ToolbarTitleItem>
+                        <ToolbarItem>
+                            <TableFilterFinalForm filterApi={filterApi}>
+                                <FilterBar>
+                                    <FilterBarField label={"Username"} name="username">
+                                        <Field name="username" type="text" component={FinalFormInput} fullWidth />
+                                    </FilterBarField>
+                                    <FilterBarMoreFilters>
+                                        <FilterBarField label={"Email"} name="email">
+                                            <ExampleWithSelect />
```

Comment:
that doesn't exist :)

(I'd remove the whole email filter, those details should be documented in the filterbar docs)

Reply:
that's true. Removed e-mail filter